sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_file_system_client.hpp

```diff
@@ -189,10 +189,10 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
      *
      * @param options Optional parameters to execute this function.
      * @param context Context for cancelling long running operations.
-     * @return A GetDataLakeFileSystemAccessPolicyResult describing the container's access policy.
+     * @return A DataLakeFileSystemAccessPolciy describing the container's access policy.
      * @remark This request is sent to blob endpoint.
      */
-    Azure::Response<Models::GetDataLakeFileSystemAccessPolicyResult> GetAccessPolicy(
+    Azure::Response<Models::DataLakeFileSystemAccessPolciy> GetAccessPolicy(
```

Comment:
typo:
Policy not Polciy
```suggestion
    Azure::Response<Models::DataLakeFileSystemAccessPolicyResult> GetAccessPolicy(
```

Reply:
same as in blob, duplicate